css-layout-classes/index.html

```diff
@@ -0,0 +1,120 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link href="style.css" rel="stylesheet" />
+    <title>Document</title>
+  </head>
+  <body>
+    <div class="container">
+      <div class="row">
+        <div class="column-full"></div>
+        <h1>Choose your character!</h1>
+      </div>
+    </div>
+    <div class="row">
+      <div class="column-one-third">
+        <input
+          type="image"
+          id="link"
+          alt="NintendoCharacterLink"
+          src="images/link.png" />
+      </div>
+      <div class="column-one-third">
+        <h2>Super Smash Bros. Ultimate</h2>
+        <p>
+          Super Smash Bros. Ultimate received universal acclaim from both
+          critics and players, with some critics calling it the best installment
+          in the series. It received praise for its large amount of content and
+          fine-tuning of existing Smash gameplay elements, although its online
+          mode was widely criticized.
+        </p>
+      </div>
+      <div class="column-one-third">
+        <input
+          type="image"
+          id="yoshi"
+          alt="NintendoCharacterYoshi"
+          src="images/yoshi.png" />
+      </div>
+      <div class="row">
+        <div class="column-one-fourth">
+          <input
+            type="image"
+            id="kirby"
+            alt="NintendoCharacterKirby"
+            src="images/kirby.png" />
+        </div>
+        <div class="column-one-fourth">
+          <input
+            type="image"
+            id="Fox"
+            alt="NintendoCharacterFox"
+            src="images/fox.png" />
+        </div>
+        <div class="column-one-fourth">
+          <input
+            type="image"
+            id="pikachu"
+            alt="NintendoCharacterPikachu"
+            src="images/pikachu.png" />
+        </div>
+        <div class="column-one-fourth">
+          <input
+            type="image"
+            id="ness"
+            alt="NintendoCharacterNess"
+            src="images/ness.png" />
+        </div>
+      </div>
+      <div class="row">
+        <div class="column-one-half">
+          <h2>Donkey Kong</h2>
+          <p>
+            Donkey Kong is a series of video games featuring the adventures of a
+            gorilla called Donkey Kong, conceived by Shigeru Miyamoto in 1981.
+            The franchise consists mainly of two games genres, but also includes
+            additional spin-off titles of various genres.
+          </p>
+          <p>
+            The games of the first genre are mostly single-screen
+            platform/action puzzle types, featuring Donkey Kong as the opponent
+            in an industrial construction setting.
+          </p>
+        </div>
```

Comment:
This should technically be a `.column-two-thirds`, because we want it to take up the majority of this row.